css/style.css

```diff
@@ -0,0 +1,544 @@
+  padding-top: 91px;
+  padding-left: 60px;
+  max-width: 195px;
+}
+
+.main__header {
+  font-family: Gotham Pro Bold, sans-serif;
+  font-size: 30px;
+  line-height: 29px;
+  text-transform: uppercase;
+  color: #FFF;
+  padding-bottom: 15px;
+}
+
+.main__link {
+  position: relative;
+  text-decoration: none;
+  font-family: Gotham Pro Bold, sans-serif;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #FFF;
+}
+
+.main__link::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 78px;
+  bottom: 2px;
+  background: url('../icons/arrow.png');
+  width: 45px;
+  height: 8px;
+}
+
+.main__subheader {
+  font-family: Gotham Pro Bold, sans-serif;
+  font-size: 22px;
+  line-height: 21px;
+  text-transform: uppercase;
+  color: #000;
+  text-align: center;
+  margin-bottom: 15px;
+}
+
+.main__card-wrapper{
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 26px;
+}
+
+.product-card {
+  position: relative;
+  max-width: 230px;
+}
+
+.product-card::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 215px;
+  bottom: 84px;
+  width: 1px;
+  height: 100px;
+  opacity: 0.2;
+  background: #000;
+}
+
+.product-card:last-child::after{
+  display: none;
+}
+
+.product-card__img {
+  padding-bottom: 23px;
+}
+
+.product-card__discr {
+  font-family: Gotham Pro Regular, sans-serif;
+  font-size: 12px;
+  line-height: 11px;
+  color: #000;
+  text-align: left;
+  padding-bottom: 5px;
+}
+
+.product-card__options {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product-card__price {
+  font-family: Gotham Pro Bold, sans-serif;
+  font-size: 14px;
+  line-height: 13px;
+  color: #000;
+}
+
+.product-card__sel {
+  width: 50px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.product-card__btn {
+  height: 7px;
+  width: 7px;
+  border-radius: 50%;
+  border: none;
+}
+
+.product-card__btn_selected {
+  height: 9px;
+  width: 9px;
+}
+
+.product-card__btn_grey {
+  background: #c4c4c4;
+}
+
+.product-card__btn_blue {
+  background: #41479B;
+}
+
+.product-card__btn_red {
+  background: #b62e3c;
+}
+
+.product-card__btn_yellow {
+  background: #fdb431;
+}
+
+.product-card__btn_brown {
+  background: #762E31;
+}
+
+.product-card__btn_orange {
+  background: #e29d69;
+}
+
+.product-card__btn_black {
+  background: #2c2c2c;  
+}
+
+/* Медиа запросы */
+
+@media (max-width: 1024px) {
+
+  body {
+  min-width: 1024px;
+  }
+
+  .container {
+    max-width: 930px;
+  }
+
+  .heading__logo {
+    margin-right: 29px;
+  }
+
+  .search-form__input {
+    margin-right: 32px;
+  }
+
+  .heading__signin {
+    margin-right: 25px;
+  }
+
+  .heading__cart-wrapper {
+    margin-right: 27px;
+  }
+
+  .main__wrapper {
+    margin-bottom: 67px;
+  }
+}
+
+@media (max-width: 768px) {
+
+  body {
+    min-width: 768px;
+  }
+
+  .heading__nav-wrapper ,
+  .heading__extras {
+    display: none;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5575166)